docs/feature_dip_switch.md

```diff
@@ -92,6 +94,20 @@ void dip_switch_update_mask_user(uint32_t state) {
 }
 ```
 
+For DIP switch processing on the slave half of split keyboards, you can add `dip_switch_scan(false);`
```

Comment:
I'm not a fan of this.  If anything, it should probably be handled like the encoders.

Reply:
ya, was on the fence about this. Maybe the correct path forward is to automatically call it on both halves, and add a define to disable it on the slave if there is negative behavior on certain keyboards due to this change?

Reply:
Well, if/when the extensibly split PR lands, code it like the matrix and encoder, handling it only when there is a change would drastically reduce the impact it would have.

Reply:
hmm, I think you are misunderstanding the note a bit. In our case we had dip switches on each half that we wanted to retain independent operation on. At least originally. So split PR or not wouldn't have impacted how we used this.

Reply:
Part of our use case here was (ab)using the dip switches as an extra switch out of the matrix. In general, I don't think people want dip switches for configuration to be used on both halves, but extensible split would make pushing configuration between halves easier. 

We can remove that line from the documentation though, since most people don't need that functionality

Reply:
Removed this note from the docs per discussion.